### PR TITLE
Enable zooming on flag submissions graph

### PR DESCRIPTION
**Pull request by camdenmoors**
This enables zooming on the flag submissions graph and adds a little explanation at the bottom.

### CODE REVIEW

File: app/views/games/summary.html.haml [hunk 1]
```diff
@@ -10,4 +10,6 @@
 - if @flags_per_hour.count > 1
   %h3.muted
     Flag Submissions
-    = line_chart @line_chart_data, colors: ['#b8d12f', '#00abca']
+    = line_chart(@line_chart_data, library: {chart: {zoomType: 'x'}}, colors: ['#b8d12f', '#00abca'])
+  %p.muted
+    = t('game.zoom_help')
```
Comment (rbclark) [inline app/views/games/summary.html.haml:15]: There is a setting in your editor to automatically add new lines to the end of a file, generally the best practice is to have that enabled: https://thoughtbot.com/blog/no-newline-at-end-of-file

Comment (camdenmoors) [reply app/views/games/summary.html.haml:15]: Got it, and enabled now, thanks.

Comment (rbclark) [reply app/views/games/summary.html.haml:15]: Can you amend this commit so the change is applied to this file as well?

Comment (camdenmoors) [reply app/views/games/summary.html.haml]: Done

**Comment by rbclark** [comment]
I just made a few small tweaks to this branch so both graphs were able to zoom and also made the grouping group by day when the competition is running for 7 or more days.

**Pull request merged**